test/integration/basic_monitor_crossconnect_metrics_test.go

```diff
@@ -18,6 +18,12 @@ import (
 
 func TestSimpleMetrics(t *testing.T) {
 	g := NewWithT(t)
+
+	if !kubetest.IsBrokeTestsEnabled() {
+		t.Skip("Temporary skip TestSimpleMetrics with vpp-agent v2.3.0")
+		return
+	}
+
```

Comment:
Why are we skipping this test?

Reply:
It seems Artyom is waiting for Denis feedback on this test.

Reply:
Nevermind... I see now.  And see we are on it.